### ISSUE TITLE
nfancurve doesn't keep fan speed as it should when set to low values <30%, speed keeps dropping to 0%

### ISSUE THREAD

**Issue by mental-advisor** [nan0s7/nfancurve#59]
By default, config file has:
```
fcurve="25 40 55 70 85" # fan speeds
tcurve="35 45 55 65 75" # temperatures
```

I tried to edit as:
```
fcurve="10 15 20 70 85" # fan speeds
tcurve="35 45 55 65 75" # temperatures
```
[GPU util and fan speed.webm](https://github.com/nan0s7/nfancurve/assets/167088385/b17ce533-6016-4db6-8e41-709c37148c84)

Readings from Vitals app. First one is GPU temp in C, second is GPU util %, third is Fan speed %.
And as you can see, the fan speed keeps dropping to 0, and then the script (I'm guessing) brings it up again to required value.
When I leave fcurve by default, the fan speeds seem to be working normally.
I feel like driver and script are conflicting with each other, with former trying to reset fan speed to 0. I'd really like to have low fan speeds when GPU temps are in between 20-40C.
Also, without the script, but with coolbits = 12 enabled where appropriate, the 10% value of fan speed also can't stay consistent in Nvidia Settings GUI app. When I set 10 to Fan 0 and Fan 1, RPM value keeps dropping to 0 and back to ~650-850 values. Like Nvidia drivers tries to set it, and then something resets it to 0. A cat and mouse game.
Through trial and error, I determined that when both fans set to 30% in Nvidia Settings ("Thermal Settings"), then the RPM values for both fans show up as more or less consistent, albeit fluctuate a bit.
I know that there seem to be a limitation on some GPU wherein they can't have less than 30%? It's like either 0% or 30%, no in between possible, why? How can I circumvent this?? Why would anyone put this! Also, vbios (automatic) doesn't spin fans even at 45C!

I have a 2000s series RTX GPU from Nvidia. Ubuntu 23.10.
NVIDIA-SMI 545.29.06              Driver Version: 545.29.06    CUDA Version: 12.3

**Comment by dev0null** [comment]
Glad to found this since I ever thought it was my card that was faulty.
I have been having this issue too and I had to set the minimum speed at 48-49% to prevent that behaviour.
I wonder if with a proprietary gpu control software (afterburner, evga x1) does the same.

**Comment by mental-advisor** [comment]
> wonder if with a proprietary gpu control software (afterburner, evga x1) does the same.

Yes, MSI Afterburner also can't keep fan speed under 30% in Windows.
Seems like manufacturer put in this limitation deliberately. I wonder if someone can flash bios/firmware of GPU to remove this limitation.
